src/displayList.js

```diff
@@ -0,0 +1,41 @@
+import loadImage from './load-image-module';
+
+const displayList = (array) => {
+  console.log('hello');
+  const listContainer = document.getElementById('list-container');
+  listContainer.innerHTML = '';
+
+  for (let j = 0; j < 10; j += 3) {
+    const row = document.createElement('div');
+    row.classList.add('row');
+
+    for (let i = 0; i < 3; i += 1) {
+      const card = document.createElement('div');
+      card.classList.add('col-sm', 'card');
```

Comment:
- [ ] I would recommend adding a **padding-top** and **padding-bottom** to the `card element` because the cards at the very top touch the header and it doesn't look very nice, and the position of the `comment button` changes depending on the **size of the name** text and the **likes**.